resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -400,6 +400,17 @@ process_host_module(main_muc_component_config, function(host_module, host)
             end
         end
 
+        -- Check for display name if missing return an error
```

Comment:
Shouldn't this be a separate module? This functionality is not exclusive to lobby is it?

Reply:
This is for lobby, we are reporting this through jiconop when lobby is enabled so we can make the display name required.
https://github.com/jitsi/jitsi-meet/blob/e8766b265d7c7fe792335aa26bfd8bc644c3469f/resources/prosody-plugins/mod_muc_lobby_rooms.lua#L269

Now when dropping initial connection on opening pre-join we are adding this if someone tries to join without a display name when lobby is enabled to fail and set the UI that display name is required, so the participant enters a dispalyname and retries.

Reply:
But this does not work if prejoin is not enabled ... the tests catch that failing case ...  need to look into